core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/ConfigImplicits.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import pureconfig.ConfigReader
+import cats.syntax.all._
+import org.broadinstitute.dsp.{ChartName, ChartVersion}
+import pureconfig.error.ExceptionThrown
+
+import java.nio.file.{Path, Paths}
+
+object ConfigImplicits {
```

Comment:
Just added because `Config.scala` is getting large? Should we move all the implicits here?

Reply:
I meant to talk about this during walkthrough but we didn't have time...But basically I'm introducing pureconfig in this PR....For new configs, I think we should just use `pureconfig`, which will be less manual. These are implicits for primitive types that can be shared by different projects (when we move multi project structure)